src/utilities.js

```diff
@@ -0,0 +1,13 @@
+const fs = require("fs");
+
+const writeToFile = (filePath, data) => {
+  try {
+    fs.writeFileSync(filePath, data);
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+module.exports = {
```

Comment:
Remember you can use your object shorthand here, since the key name is the same as the value, so we can just do:

`module.exports = { writeToFile }`